pkg/download/download.go

```diff
@@ -90,12 +90,23 @@ func Download(uri, destination string, mode os.FileMode, sha256sum []byte) (stri
 
 // Download takes an URL and the User-Agent string to use in the http request
 // it takes  http.RoundTripper as the third argument to be used by the client
```

Comment:
Can you update the comment while at it? I forgot to update it when I moved code around.

Reply:
Done